src/js/component/TodoListContainer/TodoContainer/TodoList.js

```diff
@@ -0,0 +1,93 @@
+
+function TodoList(parent, {onDelete, onToggle, onEdit}) {
+  this.$parent = parent;
+  this.todoItems = [];  
+  this.dom = {};
+  this.$todoContainer = {};
+
+  this.setDom = () => {
+    this.dom = document.createElement('ul');
+    this.dom.className = 'todo-list';
+    this.$parent.prepend(this.dom);
+  }
+
+  this.setState = updatedTodoItems => {
+    this.todoItems = updatedTodoItems;
+    this.setDom();
+    this.render(this.todoItems);
+  }
+
+  this.render = (items) => {
+    const template = items.map(item => this.template(item)).join('');
+    this.dom.innerHTML = template;
+    this.setEvent();
+  }
+
+  this.template = item => {
+    return `
+      <li class="todo-list-item ${item.isCompleted ? "completed" : ''}"  id=${item._id}>
+        <div class="view">
+          <input class="toggle" type="checkbox" id=${item._id} data-complete=${item.isCompleted}  ${item.isCompleted ? "checked" : null} />
+          <label class="label">
+            <div class="chip-container">
+              <select class="chip select">
+                <option value="0" selected>순위</option>
+                <option value="1">1순위</option>
+                <option value="2">2순위</option>
+              </select>
+            </div>
+            ${item.contents}
+          </label>
+          <button class="destroy"></button>
+        </div>
+        <input class="edit" value="${item.contents}" />
+      </li>`
+  }
+  
+  this.setEvent = () => {
+    const $destroyBtns = document.querySelectorAll('.destroy');
+    const $toggleBtns = document.querySelectorAll('.toggle');
+    const $todoLabel = document.querySelectorAll('.label');
+    const $inputEdit = document.querySelectorAll('.edit');
+    [...$destroyBtns].forEach(btn => btn.addEventListener('click', event => this.deleteTodoItem(event)));
```

Comment:
nodeList는 forEach 매서드를 사용할 수 있어요. 
혹시 배열화를 하신 이유가 있나요?
```suggestion
    $destroyBtns.forEach(btn => btn.addEventListener('click', event => this.deleteTodoItem(event)));
```